homeassistant/components/evil_genius_labs/diagnostics.py

```diff
@@ -1,6 +1,7 @@
 """Diagnostics support for Evil Genius Labs."""
 from __future__ import annotations
 
+from homeassistant.components.diagnostics.const import REDACTED
```

Comment:
can we allow importing it from the main package instead so other integrations don't need to bother about the structure of diagnostics.
```suggestion
from homeassistant.components.diagnostics import REDACTED
```

Reply:
Added a re-export